idunn/api/geocoder.py

```diff
@@ -32,6 +34,12 @@ def post_filter_intention(intention, bragi_response, limit):
             matches += 1
             if matches > expected_matches:
                 return True
+
+    logging.info(
+        "Ignored intention which doesn't match enough geocoding results `%s`",
+        intention.description.query,
+        extra={"intention": intention, "bragi_response": bragi_response, "limit": limit},
```

Comment:
`ìntention` is a Pydantic object and would appear as a `__repr__` string, `intention.dict()` should be used instead.

`bragi_response` is very verbose here. Maybe we could keep places "id" and "label" only, for debugging purposes ?

Reply:
Nice catch, it indeed seems dumb to dump the whole geocoding result :+1: